packages/@aws-cdk/aws-msk/README.md

```diff
@@ -151,6 +149,37 @@ const cluster = new msk.Cluster(this, 'cluster', {
 });
 ```
 
+
+### SASL/IAM + TLS
+
+To enable client authentication with TLS and IAM you can provide the configurations used for each separately as explained above but instead under the ´saslTls´ method:
+
```

Comment:
[nit] Docs change, references prompting for scrolling are breaking the flow, sections can switch around.

Why not have the section be stand-alone?